cdap-ui/app/features/flows/controllers/detail-ctrl.js

```diff
@@ -58,11 +59,13 @@ angular.module(PKG.name + '.feature.flows')
     });
 
     $scope.do = function(action) {
+      $scope.status = action;
```

Comment:
Say I have a flow that is 'RUNNING'.
Now, If I click stop, then the status is 'STOPPED' automatically?
What happens the very next time I poll the back end for status? (it might be 'STOPPING').